packages/vats/src/proposals/upgrade-paRegistry-proposal.js

```diff
@@ -0,0 +1,65 @@
+// @ts-check
+import { E } from '@endo/far';
+import { AmountMath } from '@agoric/ertp';
+import { Stable } from '@agoric/internal/src/tokens.js';
+
+/**
+ * @param {BootstrapPowers & {
+ *   consume: {
+ *     vatAdminSvc: VatAdminSvc;
+ *     vatStore: MapStore<
+ *       string,
+ *       import('@agoric/swingset-vat').CreateVatResults
+ *     >;
+ *   };
+ * }} powers
```

Comment:
`BootstrapPowers` already has `vatAdminSvc` and `vatStore` defined this way, doesn't it?

```suggestion
 * @param {BootstrapPowers} powers
```

Reply:
done